paths.c

```diff
@@ -721,6 +721,15 @@ enum rarch_content_type path_is_media_type(const char *path)
 
    string_to_lower(ext_lower);
 
+   /* hack, to detect livestreams so the ffmpeg core can be started */
```

Comment:
This is a hack, not sure it's the right way to do it but it works. With this you can open streams with the ffmpeg core.